src/renderer/components/+storage-volumes/volume-details-list.tsx

```diff
@@ -0,0 +1,89 @@
+import "./volume-details-list.scss";
+
+import React from "react";
+import { observer } from "mobx-react";
+import { PersistentVolume } from "../../api/endpoints/persistent-volume.api";
+import { autobind } from "../../../common/utils/autobind";
+import { TableRow } from "../table/table-row";
+import { cssNames, prevDefault } from "../../utils";
+import { showDetails } from "../kube-object/kube-object-details";
+import { TableCell } from "../table/table-cell";
+import { Spinner } from "../spinner/spinner";
+import { DrawerTitle } from "../drawer/drawer-title";
+import { Table } from "../table/table";
+import { TableHead } from "../table/table-head";
+import { volumesStore } from "./volumes.store";
+import kebabCase from "lodash/kebabCase";
+
+interface Props {
+  persistentVolumes: PersistentVolume[];
+}
+
+enum sortBy {
+  name = "name",
+  status = "status",
+  capacity = "capacity",
+}
+
+@observer
+export class VolumeDetailsList extends React.Component<Props> {
+  private sortingCallbacks = {
+    [sortBy.name]: (volume: PersistentVolume) => volume.getName(),
+    [sortBy.capacity]: (volume: PersistentVolume) => volume.getCapacity(),
+    [sortBy.status]: (volume: PersistentVolume) => volume.getStatus(),
+  };
+
+  @autobind()
+  getTableRow(uid: string) {
+    const { persistentVolumes } = this.props;
+    const volume = persistentVolumes.find(volume => volume.getId() === uid);
+
+    return (
+      <TableRow
+        key={volume.getId()}
+        sortItem={volume}
+        nowrap
+        onClick={prevDefault(() => showDetails(volume.selfLink, false))}
+      >
+        <TableCell className="name">{volume.getName()}</TableCell>
+        <TableCell className="capacity">{volume.getCapacity()}</TableCell>
+        <TableCell className={cssNames("status", kebabCase(volume.getStatus()))}>{volume.getStatus()}</TableCell>
+      </TableRow>
+    );
+  }
+
+  render() {
+    const { persistentVolumes } = this.props;
+    const virtual = persistentVolumes.length > 100;
+
+    if (!persistentVolumes.length && !volumesStore.isLoaded) return (
+      <div className="VolumeDetailsList flex justify-center"><Spinner/></div>
+    );
+    if (!persistentVolumes.length) return null;
+
+    return (
+      <div className="VolumeDetailsList flex column">
+        {<DrawerTitle title="Persistent Volumes"/>}
```

Comment:
No need to have this in `{}`, it is slightly confusing.

```suggestion
        <DrawerTitle title="Persistent Volumes"/>
```

Reply:
thank you, my bad, I'll fix it